deploy/chart/catalog_resources/rh-operators/descheduler.csv.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    categories: openshift optional
+    certifiedLevel: Primed
+    containerImage: registry.svc.ci.openshift.org/openshift/origin-v4.0:descheduler-operator
+    createdAt: 2019/11/15
+    description: An operator to run the OpenShift descheduler
+    healthIndex: B
+    longDescription: |
+      # Descheduler for Kubernetes
+
+      ## Introduction
+
+      Scheduling in Kubernetes is the process of binding pending pods to nodes, and is performed by
+      a component of Kubernetes called kube-scheduler. The scheduler's decisions, whether or where a
+      pod can or can not be scheduled, are guided by its configurable policy which comprises of set of
+      rules, called predicates and priorities. The scheduler's decisions are influenced by its view of
+      a Kubernetes cluster at that point of time when a new pod appears first time for scheduling.
+      As Kubernetes clusters are very dynamic and their state change over time, there may be desired
+      to move already running pods to some other nodes for various reasons
+
+      * Some nodes are under or over utilized.
+      * The original scheduling decision does not hold true any more, as taints or labels are added to
+      or removed from nodes, pod/node affinity requirements are not satisfied any more.
+      * Some nodes failed and their pods moved to other nodes.
+       New nodes are added to clusters.
+
+      Consequently, there might be several pods scheduled on less desired nodes in a cluster.
+      Descheduler, based on its policy, finds pods that can be moved and evicts them. Please
+      note, in current implementation, descheduler does not schedule replacement of evicted pods
+      but relies on the default scheduler for that.
+
+      ## Compatibility matrix
+
+          Descheduler  | supported Kubernetes version
```

Comment:
We have a `NativeAPIs` block in the CSV that you can use to list exactly which APIs are required by your operator to exist in kubernetes in order to run, which (in most cases) will let you avoid tracking this type of information.

Reply:
I will look into it.